### ISSUE TITLE
Bug: Ruby recipe in 6.1.2 release references old library helpers

### ISSUE THREAD

**Issue by martinisoft** [sous-chefs/postgresql#522]
### Cookbook version
6.1.2

### Chef-client version
Chef 13.8

### Platform Details
Ubuntu 16.04

### Scenario:
Running the ruby recipe produces a chef compile time error

### Steps to Reproduce:
Include the ruby recipe from the 6.1.2 release

### Expected Result:
The ruby recipe should install the pg gem into chef omnibus

### Actual Result:
Stacktrace:
```
NameError uninitialized constant Opscode::PostgresqlHelpers 
    /var/chef/cache/cookbooks/postgresql/recipes/ruby.rb:22 from_file
    gems/chef-13.5.3/lib/chef/mixin/from_file.rb:30 instance_eval
    gems/chef-13.5.3/lib/chef/mixin/from_file.rb:30 from_file
    gems/chef-13.5.3/lib/chef/cookbook_version.rb:205 load_recipe
    gems/chef-13.5.3/lib/chef/run_context.rb:343 load_recipe
    gems/chef-13.5.3/lib/chef/run_context.rb:299 block in include_recipe
    gems/chef-13.5.3/lib/chef/run_context.rb:298 each
    gems/chef-13.5.3/lib/chef/run_context.rb:298 include_recipe
    gems/chef-13.5.3/lib/chef/dsl/include_recipe.rb:26 include_recipe
    /var/chef/cache/cookbooks/dnsimple-postgres/recipes/_chef_ruby_gem.rb:43 from_file
    gems/chef-13.5.3/lib/chef/mixin/from_file.rb:30 instance_eval
    gems/chef-13.5.3/lib/chef/mixin/from_file.rb:30 from_file
    gems/chef-13.5.3/lib/chef/cookbook_version.rb:205 load_recipe
    gems/chef-13.5.3/lib/chef/run_context.rb:343 load_recipe
    gems/chef-13.5.3/lib/chef/run_context.rb:299 block in include_recipe
    gems/chef-13.5.3/lib/chef/run_context.rb:298 each
    gems/chef-13.5.3/lib/chef/run_context.rb:298 include_recipe
    gems/chef-13.5.3/lib/chef/dsl/include_recipe.rb:26 include_recipe
    /var/chef/cache/cookbooks/dnsimple-postgres/recipes/default.rb:10 from_file
    gems/chef-13.5.3/lib/chef/mixin/from_file.rb:30 instance_eval
    gems/chef-13.5.3/lib/chef/mixin/from_file.rb:30 from_file
    gems/chef-13.5.3/lib/chef/cookbook_version.rb:205 load_recipe
    gems/chef-13.5.3/lib/chef/run_context.rb:343 load_recipe
    gems/chef-13.5.3/lib/chef/run_context/cookbook_compiler.rb:163 block in compile_recipes
    gems/chef-13.5.3/lib/chef/run_context/cookbook_compiler.rb:160 each
    gems/chef-13.5.3/lib/chef/run_context/cookbook_compiler.rb:160 compile_recipes
    gems/chef-13.5.3/lib/chef/run_context/cookbook_compiler.rb:77 compile
    gems/chef-13.5.3/lib/chef/run_context.rb:192 load
    gems/chef-13.5.3/lib/chef/policy_builder/expand_node_object.rb:97 setup_run_context
    gems/chef-13.5.3/lib/chef/client.rb:513 setup_run_context
    gems/chef-13.5.3/lib/chef/client.rb:281 run
    gems/chef-13.5.3/lib/chef/application.rb:291 block in fork_chef_client
    gems/chef-13.5.3/lib/chef/application.rb:279 fork
    gems/chef-13.5.3/lib/chef/application.rb:279 fork_chef_client
    gems/chef-13.5.3/lib/chef/application.rb:244 block in run_chef_client
    gems/chef-13.5.3/lib/chef/local_mode.rb:44 with_server_connectivity
    gems/chef-13.5.3/lib/chef/application.rb:232 run_chef_client
    gems/chef-13.5.3/lib/chef/application/client.rb:469 sleep_then_run_chef_client
    gems/chef-13.5.3/lib/chef/application/client.rb:458 block in interval_run_chef_client
    gems/chef-13.5.3/lib/chef/application/client.rb:457 loop
    gems/chef-13.5.3/lib/chef/application/client.rb:457 interval_run_chef_client
    gems/chef-13.5.3/lib/chef/application/client.rb:441 run_application
    gems/chef-13.5.3/lib/chef/application.rb:59 run
    gems/chef-13.5.3/bin/chef-client:26 <top (required)>
    /usr/bin/chef-client:58 load
    /usr/bin/chef-client:58 <main>
```

**Comment by martinisoft** [comment]
So the issue is that @damacus released 6.1.2 from master instead of the 6.x branch so we should consider making one more release off 6.x and call it 6.1.3 because 6.1.2 has breaking changes and bugs.

**Comment by damacus** [comment]
Fixed in latest master release.
